docs/api-examples/resend.mdx

```diff
@@ -60,6 +60,12 @@ In order to send an email correctly, you will need to modify the boilerplate cod
       }
 ```
 
+### Add your API key
+
+If you don't already have your Resend API token in your environment, you can add it to your `.env` file as `RESEND_TOKEN=<your-resend-token>"`.
+
+You can create a new API Token from the [API Keys section](https://resend.com/api-keys) of your Resend Account.
```

Comment:
Maybe I'd add something like `Make the token has full access` because otherwise it didn't work for me.

Reply:
`Sender access` works just fine, it's just limited to send only to the address registered with the account. Works fine for me, but let me know if that's not the case for you.

Reply:
Sorry, you're right. My issue was not using the personal email 👍